### ISSUE TITLE
I am unable to use the GPU of my 3060 for inference.

### ISSUE THREAD

**Issue by c469591** [karaokenerds/python-audio-separator#11]
Hello, I tried following the instructions and installed the necessary packages for GPU acceleration using the command "pip install torch "optimum[onnxruntime-gpu]". However, when running it, I encountered an error and couldn't process with GPU acceleration.
```
2023-09-01 10:26:44.946 - INFO - cli - Separator beginning with input file: I:\02.当时的我们.flac
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\audio-separator-NjHMt13R-py3.10\Scripts\audio-separator", line 6, in <module>
    sys.exit(main())
  File "I:\分離\python-audio-separator\audio_separator\utils\cli.py", line 98, in main
    separator = Separator(
  File "I:\分離\python-audio-separator\audio_separator\separator.py", line 119, in __init__
    raise Exception("CUDA requested but not available with current Torch installation. Do you have an Nvidia GPU?")
Exception: CUDA requested but not available with current Torch installation. Do you have an Nvidia GPU?
```
I have an Nvidia graphics card 3060.
I also used poetry to deploy the virtual environment, but encountered the same error message. How can I solve this problem? I am using Windows system with Python 3.10.10. Thank you!

**Comment by beveradb** [comment]
I don't have a CUDA GPU or a Windows machine to test, but my guess would be uninstalling torch before installing it again with flags such as `pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu117 --upgrade --force-reinstall` might get you working.

If not, try googling "install torch with cuda" and see if the internet can help more than me 🤞 

If you figure it out please share your learnings here, we can update the docs to help other windows users!

**Comment by c469591** [comment]
Hello, I didn't follow the instructions in the document to install PyTorch. Instead, I used
pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu117
But recently, I happened to be researching other projects and also installed.
NVIDIA CUDA Toolkit
This is installed using the CUDA installation package found on the NVIDIA official website. 
I'm not sure if it was pip that made it work or if it was because I installed this official version of CUDA, but now I can successfully use GPU for inference. 
Thank you for providing the information.
